server/src/main/java/org/opensearch/ingest/IngestService.java

```diff
@@ -997,7 +997,7 @@ private void innerBatchExecute(
         Consumer<List<IngestDocumentWrapper>> handler
     ) {
         if (pipeline.getProcessors().isEmpty()) {
-            handler.accept(null);
+            handler.accept(toIngestDocumentWrappers(slots, indexRequests));
```

Comment:
Codecov is claiming this is not covered, which suggests your unit test isn't actually testing this case. Is that right?